build.rs

```diff
@@ -348,6 +365,36 @@ fn build() -> io::Result<()> {
     enable!(configure, "BUILD_LIB_AVS", "libavs");
     enable!(configure, "BUILD_LIB_XVID", "libxvid");
 
+    // hardware accelleration
+    enable!(configure, "BUILD_VAAPI", "vaapi");
+    enable!(configure, "BUILD_VULKAN", "vdpau");
+    enable!(configure, "BUILD_OPENCL", "vaapi");
```

Comment:
These look like typos

Reply:
Yeah thanks, I should’ve made it as a draft cause I’m still experimenting with options